static/app/views/ddm/widgetDetails.tsx

```diff
@@ -54,7 +54,13 @@ export function WidgetDetails() {
             <TabPanels.Item key={Tab.SAMPLES}>
               <SampleTable
                 mri={selectedWidget?.mri}
-                query={selectedWidget?.query}
+                query={
+                  selectedWidget?.focusedSeries
+                    ? `${selectedWidget.query} ${selectedWidget.focusedSeries
+                        .split(', ')
+                        .join(' ')}`.trim()
```

Comment:
I would advise wrapping the values of the query in `""` since it might be that certain groups names are not supported by the unquoted strings.

Reply:
good point! fixed in https://github.com/getsentry/sentry/pull/63423/commits/295d7ea166c95673135996a0b5e4bad525c572e3